mypy/semanal.py

```diff
@@ -3283,6 +3291,9 @@ def visit_name_expr(self, expr: NameExpr) -> None:
                 self.fail("'{}' is a type variable and only valid in type "
                           "context".format(expr.name), expr)
             else:
+                if n.is_aliasing:
```

Comment:
These 3 lines are repeated 3 times. Would it make sense to move them to a utility method?